src/option-mapping.json

```diff
@@ -44,7 +45,7 @@
     },
     {
         "actionOption": "noRemoveCollidingTunnels",
-        "dockerOption": "tunnel-pool",
+        "dockerOption": "no-remove-colliding-tunnels",
```

Comment:
same for `tunnelIdentifier -> tunnel-identifier`?

Reply:
oh, right. tnx!